sega_genesis_checksum_utility_for_python_3.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python
```

Comment:
I just noticed, this adds a new file with the `_for_python_3` suffix.

Since python2 is no longer being supported, can we just update the existing file. I don't see any benefits in maintaining both versions. (In other words, rename your new file back to the original file name `sega_genesis_checksum_utility.py`). This will also update the pull request diff to show only the updated lines.